web/src/js/__tests__/ducks/utils/storeSpec.tsx

```diff
@@ -13,9 +13,9 @@ describe("store reducer", () => {
     });
 
     it("should handle add action", () => {
-        let a = { id: "1" },
-            b = { id: "9" },
-            state = reduce(undefined, {});
+        const a = { id: "1" },
+            b = { id: "9" };
```

Comment:
Ew, what happened to the indentation? Is that really what prettier does? I personally forbid multi-declarations and would just have two `const` lines.

Reply:
I think so yes